chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -64,7 +64,10 @@ We can represent each pixel value with a single scalar, giving us four features
 Further, let's assume that each image belongs to one among the categories "cat", "chicken" and "dog".
 -->
 
-*dịch đoạn phía trên*
+Để khởi động, ta hãy bắt đầu với bài toán phân lớp hình ảnh đơn giản trước.
+Ở đây, mỗi input bao gồm một ảnh xám $2\times2$.
+Ta có thể biểu diễn mỗi giá trị điểm ảnh là một số vô hướng duy nhất, cung cấp cho ta bốn đặc trưng $x_1, x_2, x_3, x_4$.
+Hơn nữa, hãy giả rử rằng mỗi hình ảnh đều thuộc về một trong số danh mục "cat", "chiken" và "chó".
```

Comment:
chiken (typo) -> chicken
"cat", "chicken" and "chó" mình nghĩ một là dịch hết 2 là để nguyên "cat", "chicken", "dog"

Reply:
Vâng ạ, cảm ơn anh đã review bài em ạ, chỗ đó em gõ nhầm chữ "dog".